models/migrations/v87.go

```diff
@@ -0,0 +1,42 @@
+// Copyright 2017 Gitea. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package migrations
+
+import (
+	"fmt"
+
+	"github.com/go-xorm/xorm"
+)
+
+func addAvatarFieldToRepository(x *xorm.Engine) (err error) {
+
+	sess := x.NewSession()
+	defer sess.Close()
+
+	if err := sess.Begin(); err != nil {
+		return err
+	}
+
+	dialect := x.Dialect().DriverName()
+
+	switch dialect {
+	case "mysql":
```

Comment:
I need to use case, Xorm can handle this. You can use Sync2 method

Reply:
Done